package.json

```diff
@@ -50,6 +50,7 @@
     "@mux/upchunk": "^1.0.6",
```

Comment:
@erikpena want to go ahead and bump upchunk to the latest (`"^2.2"`)?  There's no breaking changes that would affect this plugin.